src/MCPClient/lib/clientScripts/create_transfer_mets.py

```diff
@@ -376,6 +601,74 @@ def event_to_premis(event):
     )
 
 
+def rights_to_premis(rights, file_uuid):
+    """
+    Converts an RightsStatement model to a PREMIS event object via metsrw.
+
+    Returns:
+        lxml.etree._Element
+    """
+    VALID_RIGHTS_BASES = ["copyright", "institutional policy", "license", "statute"]
```

Comment:
I'd prefer to move this up to module level, but` ensure_no_mutable_globals` complains (even if I make this an _immutable_ `frozenset`, lol). I don't think thats a valid complaint, but I'll submit an issue for that.